### ISSUE TITLE
Foreground icon color is black on dark blue. Impossible to see.

### ISSUE THREAD

**Issue by greazer** [microsoft/vscode-tools-for-ai#1628]
<img width="332" alt="image" src="https://user-images.githubusercontent.com/11685408/182949714-36a68b00-2394-4ede-8731-459e3fb72b62.png">

**Comment by tbombach** [comment]
Closed with this PR: https://github.com/microsoft/vscode-azureml/pull/411
